CHANGELOG.md

```diff
@@ -1,11 +1,7 @@
 # Unreleased
 
 ### Fixes
-* Updated `stripe-ios` to 23.32.\*
-
-### Fixes
-* Updated `stripe-ios` to 23.30.\*
-* Updated `stripe-android` to 20.52.\*
```

Comment:
keep the stripe android change in the changleog

Reply:
Argh, missed this before merging. Will update this back when I release the new version tag!